src/client/src/ui/notification/tradeNotificationView.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import './notification.scss';
+import classnames from 'classnames';
+
+const DONE = 'Done';
```

Comment:
I believe this is defined in `TradeNotification`, we should lift it from both places, or just put it here as suggested before.